debug-scripts/osx-debug.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+openocd -f interface/stlink-v2.cfg -f target/stm32f2x.cfg
```

Comment:
Do we actually need a one-line shell script or it could be inlined in Makefile?

Reply:
Yes, because this is the only command we need to run in defferent terminal window and there is no different way to do it